frameworks/PHP/ubiquity/ubiquity-swoole.dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM php:7.3
 
-ENV SWOOLE_VERSION=4.4.5
+ENV SWOOLE_VERSION=4.4.7
```

Comment:
Why not use the pecl install, will be faster and updated. 
Like do hamlet:
https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/PHP/hamlet/hamlet-swoole.dockerfile#L3-L5

Reply:
Also you can install a specific version:
`pecl install swoole-4.4.7`

Reply:
Well done!
ok